<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="GraphicsView" name="graphicsView"/>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Selection</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QGroupBox" name="groupBox_6">
           <property name="title">
            <string>Scale and tunning</string>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Root note</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="comboBoxRoot"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Scale name</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="comboBoxScale"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Tuning</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QComboBox" name="comboBoxTuning"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Note representation</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QRadioButton" name="radioButtonBoxPattern">
              <property name="text">
               <string>Box pattern view</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButtonThreeNotes">
              <property name="text">
               <string>Three notes per string</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButtonFourNotes">
              <property name="text">
               <string>Four notes per string</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButtonOneString">
              <property name="text">
               <string>One string</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Controls</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>View</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QPushButton" name="pushButtonFirst">
              <property name="text">
               <string>Show first</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QPushButton" name="pushButtonPrev">
                <property name="text">
                 <string>Previous</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonNext">
                <property name="text">
                 <string>Next</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonAll">
              <property name="text">
               <string>Show all</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_5">
           <property name="title">
            <string>Animation</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QPushButton" name="pushButtonStartAnim">
              <property name="text">
               <string>Start animation</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonStopAnim">
              <property name="text">
               <string>Stop animation</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonExit">
           <property name="text">
            <string>Exit</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave_To_File"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSave_To_File">
   <property name="text">
    <string>Save To File</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GraphicsView</class>
   <extends>QGraphicsView</extends>
   <header location="global">graphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonFirst</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>showFirst()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>685</x>
     <y>314</y>
    </hint>
    <hint type="destinationlabel">
     <x>888</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonExit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>681</x>
     <y>526</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonNext</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>showNext()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>723</x>
     <y>348</y>
    </hint>
    <hint type="destinationlabel">
     <x>453</x>
     <y>410</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonPrev</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>showPrev()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>635</x>
     <y>351</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonAll</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>showAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>632</x>
     <y>386</y>
    </hint>
    <hint type="destinationlabel">
     <x>474</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonStartAnim</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>startAnimation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>617</x>
     <y>469</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>460</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonStopAnim</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>stopAnimation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>629</x>
     <y>499</y>
    </hint>
    <hint type="destinationlabel">
     <x>376</x>
     <y>497</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showFirst()</slot>
  <slot>showNext()</slot>
  <slot>showPrev()</slot>
  <slot>showAll()</slot>
  <slot>startAnimation()</slot>
  <slot>stopAnimation()</slot>
 </slots>
</ui>
